scout/server/blueprints/variants/templates/variants/utils.html

```diff
@@ -319,7 +319,7 @@
           aria-expanded="false"
           aria-controls="collapseFilters">
     <strong class="me-2">Filters</strong>
-    <i class="fa fa-chevron-down chevron-icon"></i>
+    <i class="fa fa-chevron-right chevron-icon"></i>
```

Comment:
I seem to remember old code factor complaints about using italic tags for icons, and swapping them for span instead but it seems to have stopped, so who am I to question it! 😁 All good.